lib/hx509/req.c

```diff
@@ -1515,6 +1666,14 @@ hx509_request_print(hx509_context context, hx509_request req, FILE *f)
             break;
         }
     }
+    if (req->nunsupported_crit) {
+        fprintf(f, "  unsupported_critical_extensions_count: %u\n",
+                (unsigned)req->nunsupported_crit);
+    }
+    if (req->nunsupported_crit) {
```

Comment:
Coverity noticed that this looks like a copy‐paste error — `req->nunsupported_crit` should presumably read `req->nunsupported_opt`.